README.md

```diff
@@ -1,2 +1,179 @@
-# Hello-world
-First Microverse project day 2 - Set up a 'Hello Microverse' project
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="nameLogo.jpg" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Your logo isn't displaying. Please add logo properly or remove the logo entirely if you don't have one.